client/src/pages/lessons/edit.tsx

```diff
@@ -499,6 +499,36 @@ const LessonEdit = (props: {
               disabled={true}
             />
           </Grid>
+          <Grid item xs={12} md={6}>
+            <FormControl className={classes.selectForm} variant="outlined">
+              <InputLabel shrink id="classifier-arch-label">
+                Classifier Architecture
+              </InputLabel>
+              <Select
+                data-cy="classifier-arch"
+                labelId="classifier-arch-label"
+                value={
+                  lessonUnderEdit.lesson?.arch || "opentutor_classifier.lr2"
+                }
+                onChange={(e: SelectChangeEvent<string>) => {
+                  setLesson(
+                    {
+                      ...(lessonUnderEdit.lesson || newLesson),
+                      arch:
+                        (e.target.value as string) ||
+                        "opentutor_classifier.lr2",
```

Comment:
since this hardcoded value is starting to appear in multiple places, may be worth turning it into a constant